test/files/qty000/generate.py

```diff
@@ -0,0 +1,24 @@
+import ifcopenshell
+import ifcopenshell.template
+import os
+
+rule_code = "qty000"
+
+abs_path = os.path.join(os.getcwd(), "test", "files", rule_code)
+save_ifc_file = lambda file, filename: file.write(os.path.join(abs_path, filename))
+
+f = ifcopenshell.template.create(schema_identifier="IFC4X3_ADD2")
+
+save_ifc_file(f, 'pass-qty000-not_activated_no_quantity.ifc')
+
+qt = f.createIfcElementQuantity()
+door = f.createIfcDoor()
```

Comment:
I've also modified the testfiles so that they all pass the schema and syntax check
https://github.com/buildingSMART/ifc-gherkin-rules/commit/b695316aff4353c2959f8e21092f5b2c0bd69ecb